libiberty is part of GCC and is provided only statically on Slackware. We
patch CMake to use it.
Patches CMake to avoid downloading during the build since we use prefetched
sources.
--- a/third-party/folly/CMakeLists.txt	2021-04-22 08:09:46.000000000 +0200
+++ b/third-party/folly/CMakeLists.txt	2021-05-03 07:43:23.777873092 +0200
@@ -298,3 +298,6 @@
   # Just assume we have sched.h
   target_compile_definitions(folly PUBLIC "FOLLY_HAVE_SCHED_H=1")
 endif()
+
+target_link_libraries(folly /usr/lib64/libiberty.a)
+target_include_directories(folly PUBLIC /usr/include/libiberty)
--- a/third-party/brotli/CMakeLists.txt	2021-04-22 08:09:46.000000000 +0200
+++ b/third-party/brotli/CMakeLists.txt	2021-05-03 06:21:46.202704488 +0200
@@ -6,8 +6,6 @@
 # version.
 ExternalProject_Add(
   bundled_brotli
-  GIT_REPOSITORY https://github.com/google/brotli
-  GIT_TAG 4b5771bee7995d3d606221caa455c044d80434df
   CMAKE_ARGS
     -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>
     -DCMAKE_INSTALL_INCLUDEDIR=include
